src/components/OmouComponents/IdleLogout.js

```diff
@@ -131,44 +146,37 @@ const IdleLogout = () => {
             }, 900000);
         });
     };
-
-    useEffect(() => {
-        (async () => {
-            await refreshTokenAfter15Minutes();
-        })();
-
-    }, []);
     
+
     const logoutAndCloseModal = () => {
         handleClose();
         handleLogout();
     };
 
+    // Opens the modal, resets the modal timeout ref
     async function handleOpen() {
         setOpenModal(true);
-        handleReset();
-        setRemainingMsUntilPrompt(modalTimeout);
-
-        sessionTimeoutRef.current = setTimeout(logoutAndCloseModal, 5 * 1000);
+        sessionTimeoutRef.current = setTimeout(logoutAndCloseModal, modalTimeout);
     }
     
+    // Close the modal, clear the modal timeout ref, reset the token
     const handleClose = () => {
         clearInterval(sessionTimeoutRef.current);
         setOpenModal(false);
         resetToken();
     };
 
     const {
-        reset,
         getRemainingTime,
     } = useIdleTimer({
         "timeout": idleTimeout
     });
     
-    const handleReset = () => reset();
-    
+    // Resets token every 15 minutes and sets the intervals for the idle-timeout function
```

Comment:
Be descriptive as to why you're doing this - my understanding is that you're doing this to kick off the 15 minute on component load so it will keep refreshing the token every 15 minutes as long as the idleTimer never hits 18 minutes?